### PR TITLE
zh-TW: Translate async.md and async/async-await.md

### PR DESCRIPTION
**Pull request by wdzeng**
Part of #684.

### CODE REVIEW

File: po/zh-TW.po [hunk 1]
```diff
@@ -15018,16 +15020,20 @@ msgid ""
 "   runtime implements the event loop, so many of the details of Promise\n"
 "   resolution are hidden."
 msgstr ""
+"* Python 有一個類似的模型 `asyncio`。不過 `asyncio` 的 `Future` 類型是根據回呼函數而非輪詢。非同步的 Python 程式需要「迴圈（loop）」來處理，這裡的迴圈類似於 Rust 的輪詢。\n"
```
Comment (mingyc) [inline po/zh-TW.po:15023]: > 回呼函數
回呼函數 (callback)

**Review by mingyc — DISMISSED** [review]
LGTM with a few nits

**Comment by wdzeng** [comment]
Thanks for the review. All comments are fixed.

**Comment by mgeisler** [comment]
I applied the changes suggested by @hueich. Does this look okay to merge now?

**Review by mgeisler — APPROVED** [review]
Changes requested by @mingyc and @hueich have been applied, so it should be good to go.

**Pull request merged**